com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/resources/jsweb/dmloader.js

```diff
@@ -470,7 +472,9 @@ var GameArchiveLoader = {
             EngineLoader.loadAsmJsAsync(exeName);
             totalSize += EngineLoader.asmjs_size;
         }
-        this.downloadContent();
+        if (!Module['isDMFSSupported']) {
```

Comment:
Why is it done like this? Why can't we always call downloadContent() here?

Reply:
We need to get the _wasm.js first

Reply:
Ok, got it. Thanks.